docs/NativeCodeGeneration.md

```diff
@@ -73,13 +77,13 @@ driver skeleton.
 
 All Windows Drivers need to export a static function labelled &quot;DriverEntry&quot; that is invoked by the OS
 (Operating System) during loading of the driver. In addition to the standard driver related functionality, the driver
-entry skeleton would also register three [Network Module Registrar (NMR)](https://docs.microsoft.com/en-us/windows-hardware/drivers/network/introduction-to-the-network-module-registrar)
+entry skeleton also registers three [Network Module Registrar (NMR)](https://docs.microsoft.com/en-us/windows-hardware/drivers/network/introduction-to-the-network-module-registrar)
 contracts exposed by the other framework components (to export the program to the eBPF runtime, to create any maps
 required, and to import the helper functions).
 
 ## Global entry point for generated code
 
-Every generated C file will contain a single global entry point of type metadata_table_t:
+Every generated C file contains a single global entry point of type metadata_table_t:
 
 ```
 typedef struct _metadata_table
```

Comment:
This is obsolete now. Metadata table now looks like following:

` typedef struct _metadata_table
 {
        void (*programs)(program_entry_t** programs, size_t* count);
        void (*maps)(map_entry_t** maps, size_t* count);
    } metadata_table_t;`

Reply:
Thanks, updated.